addons/makeslave.pm

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/perl
+
+use Config::IniFiles;
+
+my %ini;
+
+tie %ini, 'Config::IniFiles';
+
+my $pfconf = "/usr/local/pf/conf/pf.conf";
+
+my %inipfconf;
+tie %inipfconf, 'Config::IniFiles', ( -file => "$pfconf" );
+
+my $replication_username = $inipfconf{"active_active"}{"galera_replication_username"};
+my $replication_password = $inipfconf{"active_active"}{"galera_replication_password"};
+
+
+print "Enter the MySQL root password: ";
+my $mysql_root_password = <STDIN>;
```

Comment:
Password should be ignored if defaults file exists and contains entries.

Default location: ~/.my.cnf

Expected format:
```
[client]
user=root
password=password
```